src/test/suite/fixtures/recorded/surroundingPair/textual/chuckPairHarp.yml

```diff
@@ -0,0 +1,29 @@
+languageId: plaintext
```

Comment:
Testing that `processToken` still causes whitespace cleanup for surrounding pair scopes